File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)

{

float change;

    do
    { 
      change = get_float("what is your change?\n");
} while (change < 0);

    int coins = round(change * 100);

    int q = coins / 25;
   
    int afterQuarters = coins % 25;
    
    int d = afterQuarters / 10; 

    int afterDimes = afterQuarters % 10; 
    
    int n = afterDimes / 5; 
    
    int afterNickels = afterDimes % 5; 
    
    int p = afterNickels / 1;
    
    int totalCoins = q + d + n + p; 
    
    printf("%i\n", totalCoins);
  
}

File: pset6/cash.py
from cs50 import get_float
import math

dollars = get_float("what is your change?\n")

change = (dollars * 100)

q = change // 25

afterQuarters = change % 25

d = afterQuarters // 10

afterDimes = afterQuarters % 10

n = afterDimes // 5
    
afterNickels = afterDimes % 5

p = afterNickels // 1
    
totalCoins = math.floor(q + d + n + p)

print(totalCoins)